modelx/modelx.go

```diff
@@ -692,6 +717,16 @@ type MeasurementRoot struct {
 	// Beginning is the "zero" used to compute the elapsed time.
 	Beginning time.Time
 
+	// ErrDNSBogon is the kind of error that you would like this
+	// library to return when a bogon IP address is found. The
+	// default value, nil, causes this library to only record the
+	// occurrence of such bogon in the scoreboard, but does not
+	// otherwise cause any failure. Setting this field to non-nil
+	// error causes the library instead fail when a bogon has
+	// been detected. The best value with which to initialize this
+	// field is the ErrDNSBogon variable in this package.
+	ErrDNSBogon error
```

Comment:
Do you really need this? Why not simplify and always return the package ErrDNSBogon error?